### PR TITLE
Added FPGA-Addon-Specialty-IO submodule

### PR DESCRIPTION
**Pull request by zszilagy**
- [X] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/niveristand-fpga-addon-custom-device/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Added FPGA-Addon-Specialty-IO submodule

### Why should this Pull Request be merged?

Currently the build system is down because of the missing submodule.

### What testing has been done?

Run a build on the draft pull request and it passed.

### CODE REVIEW

File: build.toml [hunk 1]
```diff
@@ -53,22 +47,9 @@ project = '{cd}'
...
-type = 'lvBuildSpec'
-project = '{api}'
-target = 'My Computer'
-build_spec = 'System Definition API'
-
 [package]
 type = 'nipkg'
+payload_dir = 'Built'
 name = 'ni-fpga-addon-{veristand_version}'
-dev_xml_path = 'Source\Addon\Custom Device FPGA Addon.xml'
```
Comment (bogdanp-ni) [inline build.toml:73]: Why is this line removed?

Comment (zszilagy) [reply build.toml:73]: The build.toml file was a copy from the original repo with incorrect paths and tags. 
I checked in the official toml documentation and the tag dev_xml_path does not appear as a valid tag for building packages.(for more details please see https://github.com/ni/niveristand-custom-device-build-tools/blob/master/docs/Toml%20Help.md)
